### PR TITLE
Allow missing or empty downtime ID

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: src/schema/rgdowntime.xsd [hunk 1]
```diff
@@ -5,7 +5,7 @@
       <xsd:element minOccurs="0" maxOccurs="unbounded" name="Downtime">
         <xsd:complexType>
           <xsd:sequence>
-            <xsd:element name="ID" type="xsd:int" />
+            <xsd:element minOccurs="0" name="ID"/>
```
Comment (brianhlin) [inline src/schema/rgdowntime.xsd]: If ID is set, should we still make sure that it's an int?

Comment (matyasselmeci) [reply src/schema/rgdowntime.xsd]: No because it might be present but empty -- as is currently the case.

Comment (matyasselmeci) [reply src/schema/rgdowntime.xsd]: Actually maybe I should remove the minOccurs="0"

Comment (brianhlin) [reply src/schema/rgdowntime.xsd]: Arghhh. Does the `rg_reader` always set an ID?

Comment (matyasselmeci) [reply src/schema/rgdowntime.xsd]: Yep. Turns out our clients don't like missing fields, even if the fields our empty.

Comment (brianhlin) [reply src/schema/rgdowntime.xsd]: When you say "even if the fields are empty", do you mean that our consumers are ok with the field being empty?

**Comment by brianhlin** [comment]
We're waiting on some answers from AGIS/SAM developers about their usage of downtime IDs

**Pull request closed without merging**